### PR TITLE
[receiver/dockerstats] Disable deprecated cpu metric

### PR DESCRIPTION
**Pull request by carlossscastro**
2nd step for the deprecation of `container.cpu.percent`

According to the deprecation plan in the [docs](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.79.0/receiver/dockerstatsreceiver#transition-to-cpu-utilization-metric-name-aligned-with-opentelemetry-specification), this PR disables the old metric by default, to be released in v0.83.0

tracking issue: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21807

### CODE REVIEW

**Comment by sigilioso** [comment]
I'm afraid the deprecation notes are not true anymore because it couldn't be merged on time.

@jamesmoessis , @rmfitzpatrick  should we update the corresponding notes or should we proceed in a different way?

**Comment by rmfitzpatrick** [comment]
I think updating the notes for disabling in the 0.87.0 release and removing in 0.88.0 makes sense given the shifted window. Apologies for missing the thread on this.

**Comment by sigilioso** [comment]
> I think updating the notes for disabling in the 0.87.0 release and removing in 0.88.0 makes sense given the shifted window. Apologies for missing the thread on this.

No worries! 🙂 I've just updated the notes.

File: receiver/dockerstatsreceiver/metadata.yaml [hunk 1]
```diff
@@ -132,19 +132,20 @@ metrics:
...
   container.cpu.percent:
-    enabled: true
-    description: "Deprecated: use `container.cpu.utilization` metric instead. Percent of CPU used by the container."
+    enabled: false
+    description: "[DEPRECATED] Use `container.cpu.utilization` metric instead. Percent of CPU used by the container."
     unit: "1"
     warnings:
-      if_enabled: This metric will be disabled in v0.82.0 and removed in v0.85.0.
+      if_configured: >-
+        The metric is deprecated and will be removed in v0.86.0. Please use `container.cpu.utilization` instead. See
```
Comment (rmfitzpatrick) [inline receiver/dockerstatsreceiver/metadata.yaml:146]:
```suggestion
        The metric is deprecated and will be removed in v0.88.0. Please use `container.cpu.utilization` instead. See
```

**Review by rmfitzpatrick — APPROVED** [review]
lgtm w/ version correction and mdatagen

File: receiver/dockerstatsreceiver/metadata.yaml (hunk 1 above)
Comment (sigilioso) [reply receiver/dockerstatsreceiver/metadata.yaml]: Sorry I forgot the metadata's version correction 🤦. I's properly updated here: [1f4c199](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/24183/commits/1f4c199b760232f29f37d350c14a9560ff41737f). Thanks!

**Review by dmitryax — APPROVED** [review]

**Pull request merged**